handlers.go

```diff
@@ -3,11 +3,25 @@ package main
 import (
 	"net/http"
 
+	"github.com/gin-contrib/sessions"
 	"github.com/gin-gonic/gin"
 )
 
 func showIndex(c *gin.Context) {
+	var score int
+
 	flags := getAllFlags()
+	session := sessions.Default(c)
```

Comment:
This one took me a while to understand, seems like in order to use `sessions.DefaultMany(c, sessionName)` the initial session created has to be with **Many**, that is `router.Use(sessions.SessionsMany(sessionNames, store))` otherwise using the current `router.Use(sessions.Sessions(defaultSessionName, store))` only creates a single session that can only be retrieved with `sessions.Default`. At this point I think is still too early to add more sessions, so for now I'll keep using `Default` to retrieve the session.